application/F3DStarter.cxx

```diff
@@ -313,6 +314,27 @@ int F3DStarter::Start(int argc, char** argv)
     }
 #endif
   }
+
+  // Parse colormap
+  if (!this->Internals->AppOptions.ColorMapFile.empty())
```

Comment:
should we check if colormap option is not empty ?

Reply:
Idk... Which one should take precedence?

Reply:
in the doc you say that colormap takes precedence, is it not implemented like that ?

Reply:
I have updated the doc to clearly specify `--colormap-file` has precedence over `--colormap`